submissions/chabuChabu/phpDatabase/viewCustomers.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+require_once("connection.php");
+
+$cookieName = 'customerList';
+
+// 1. check if cookie exists
+if (! empty($_COOKIE[$cookieName])) {
+    // 2. get cookie data
+    $result = unserialize($_COOKIE[$cookieName]);
+}
+else {
+    // 3. there is no cookie, load data from database
+    $sql = 'select id,concat(first_name," ",last_name )as full_name,email,created_at from customers';
+    $stmt = $pdo->prepare($sql);
+    $stmt->execute();
+    $result = $stmt->fetchAll();
+    //4. save in cookie
+    setcookie($cookieName, serialize($result),time() + 3600,"/","",0);
+}
+
+// 5. display result using $result only
+if (!empty($result)) {
+    echo"<h1>List of Customers</h1>";
+    echo"<table>";
+       echo"<tr>";
+            echo"<th>S/N</th>";
+            echo"<th>Full Name</th>";
+            echo"<th>Email Address</th>";
+            echo"<th>Created At</th>";
+            echo"<th>Action</th>";
+       echo"</tr>";
+       
+       foreach($result as $row){
+        echo"<tr>";
+           echo"<td>{$row['id']}</td>";
+           echo"<td>{$row['full_name']}</td>";
```

Comment:
there is no column on the `customsers` table with the name `full_name`

Reply:
@adaugochi I don't have a column in customers table called full_nane,what  have is first-name and last-name.But what i understand is that the task requires me to display these as Full Name on the browser and as a result am concatenating first_name and last_name..is my understanding wrong?

Reply:
I can't see where you are doing the concatenation

Reply:
@adaugochi i have done it in the select query